File: phasegateone/Ascending_order.py
number1 = input("First number")
number2 = input("Second number")
number3 = input("Third number")
if number1>number2 and number2>number3:
	print(number3,number2,number1)

if number2>number1 and number1>number3:
	print(number3,number1,number2)

if number3>number1 and number1>number2:
	print(number2,number1,number3)

if number1>number3 and number3>number2:
	print(number2,number3,number1)

if number2>number3 and number3>number1:
	print(number1,number3,number2)

if number3>number2 and number2>number1:
	print(number1,number2,number3 1k)

number4 = sorted((number1, number2, number3))
print(number4)

File: phasegateone/AscendingOrder.java
import java.util.Scanner;
public class AscendingOrder{
	public static void main(String[] args){
		Scanner input = new Scanner(System.in);

		System.out.println("First number");
		int number1 = input.nextInt();

		System.out.println("Second number");
		int number2 = input.nextInt();
 
		System.out.println("Third number");
		int number3 = input.nextInt(); 

		if (number1>number2 && number2>number3){
			System.out.println(number3);
			System.out.println(number2);
			System.out.println(number1);

		}else if(number2>number1 && number1>number3){
			System.out.println(number3);
			System.out.println(number1);
			System.out.println(number2);

		}else if(number3>number1 && number1>number2){
			System.out.println(number2);
			System.out.println(number1);
			System.out.println(number3);

		}else if(number1>number3 && number3>number2){
			System.out.println(number2);
			System.out.println(number3);
			System.out.println(number1);

		}else if(number2>number3 && number3>number1){
			System.out.println(number1);
			System.out.println(number3);
			System.out.println(number2);

		}else if(number3>number2 && number2>number1){
			System.out.println(number1);								System.out.println(number2);
			System.out.println(number3);
		}
	}
}